src/components/Footer.jsx

```diff
@@ -0,0 +1,17 @@
+import { Box, Text } from "@chakra-ui/react";
+
+export function Footer() {
+  return (
+    <>
+      <Box
+        bg="white"
+        boxShadow="-4px -4px 4px rgba(0, 0, 0, 0.25)"
+        padding="15px"
+        mt="35px"
+      >
```

Comment:
1. Footer should be in "footer" tag due to accessibility. You can use "as" prop on Box component.
2. Try to avoid px (absolute unit) - chakra ui provides its own spacing system (look at https://chakra-ui.com/docs/styled-system/theming/theme#spacing). And if you cannot use chakra spacing, then always use relative units like rem, em.